src/utils/store-odp-data.js

```diff
@@ -0,0 +1,150 @@
+const redis = require('redis')
+const request = require('request-promise-native')
+const { promisify } = require('util')
+
+async function redisClientConnect() {
+  return redis.createClient({
```

Comment:
We won't need a second Redis client when we do this. It should be possible to inject the original client into whatever app-level storage layer you create (your version of change-writer).

Reply:
Removed this functionality